scripts/content-modules/adjust-pages.pl

```diff
@@ -95,12 +95,22 @@ ($$$)
   return 0;
 }
 
-# sub patchSpec_because_of_SemConv_AttrRegRefactoring() {
-#   return unless $ARGV =~ /^tmp\/otel\/specification\//
-#     && applyPatchOrPrintMsgIf('2025-05-13-attribute-registry-refactoring', 'spec', '1.45.0-dev');
+sub patchSpec_because_of_SemConv_MetricReqLevelHashDNE() {
+  return unless $ARGV =~ /^tmp\/semconv\/docs\//
+    && applyPatchOrPrintMsgIf('2025-08-28-metric-request-level-hash-dne', 'semconv', '1.37.0-dev');
 
-#   s|/attributes-registry/|/registry/attributes/|g;
-# }
+  # See https://github.com/open-telemetry/semantic-conventions/issues/2690#issuecomment-3235079573
+  s|/docs/general/metrics.md#metric-requirement-levels|/docs/general/attribute-requirement-level.md#recommended|g;
```

Comment:
this one should be 
```suggestion
  s|/docs/general/metrics.md#metric-requirement-levels|/docs/general/metric-requirement-level.md#recommended|g;
```

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#recommended